### ISSUE TITLE
log file is still being used by "another" process

### ISSUE THREAD

**Issue by dexter-xyz** [pizzaboxer/bloxstrap#212]
this error appears the second time i launch roblox (it does actually open roblox just fine)
![image](https://user-images.githubusercontent.com/39269156/236598976-38387ea4-86ef-4bb2-88b1-13f5407bcf9d.png)

**Comment by EasternBloxxer** [comment]
Temporary fix would be closing bloxstrap with task manager

**Comment by pizzaboxer** [comment]
I need to get a clearer idea on where exactly this is happening. The next time this happens, get the latest log from the Bloxstrap Logs folder and send it here.

**Comment by dexter-xyz** [comment]
[Bloxstrap_20230507T111252Z.log](https://github.com/pizzaboxer/bloxstrap/files/11414833/Bloxstrap_20230507T111252Z.log)

**Comment by pizzaboxer** [comment]
I just realized that the error isn't going to be written to the log file if the error is about not being able to access it...

Here, go to `%localappdata%\Bloxstrap` in your file browser, and replace Bloxstrap.exe in the folder [with this](
https://pizzaboxer.xyz/files/bloxstrap-debug/212/Bloxstrap.exe). It should contain more verbose error messages, so post those instead the next time it happens.

**Comment by dexter-xyz** [comment]
![image](https://user-images.githubusercontent.com/39269156/236797748-42ec704b-7a1e-4b3a-897d-874f45cd9ef5.png)
[Bloxstrap_20230508T100920Z.log](https://github.com/pizzaboxer/bloxstrap/files/11420163/Bloxstrap_20230508T100920Z.log)

**Comment by pizzaboxer** [comment]
When this happens, can you open Resource Monitor, go to the Disk tab, go to the Disk Activity section, and see what process is occupying the file mentioned?

**Comment by coolpeter98** [comment]
same error

**Comment by pizzaboxer** [comment]
Possibly caused by the recent update I made that cleans out old log files? If there's anyone active here who's regularly having this problem, let me know, I can get you to try out something.

**Comment by Boi2021Miceoos** [comment]
me

**Comment by pizzaboxer** [comment]
When did you start using it?

**Comment by pizzaboxer** [comment]
Okay, thanks to some more info from @dexter-xyz I was able to reproduce it. He uses [RoGold](https://rogold.live/), and he uses one of it features which allows pinning games to the homepage. Launching a pinned game causes the error, which I was able to reproduce.

![image](https://github.com/pizzaboxer/bloxstrap/assets/41478239/ffe800af-e834-4679-a761-1f786bbac130)

Turns out that RoGold has a bug where trying to launch a game through it actually ends up launching Roblox twice. Here's it happening with the stock launcher too, except it's not a problem since it's actually able to handle something unexpected like this.

![image](https://github.com/pizzaboxer/bloxstrap/assets/41478239/ed8a8fb2-fad5-44f8-8e90-6ebcb163f928)

I'll report this to RoGold, and I'll improve Bloxstrap's handling of this.
